### ISSUE TITLE
Codecov badge in README shows 0% coverage

### ISSUE THREAD

**Issue by stefanv** [scikit-image/scikit-image#6278]
Proposed solution? Remove.

**Comment by grlee77** [comment]
Yeah, I did try briefly one day to fix it, but did not succeed and never got back to it. Let's remove it and we can always add it back later if desired.
